BridgeSDK/BridgeAPI/BridgeAPITypes/MachineGeneratedDoNotEdit/_SBBActivity.h

```diff
@@ -51,10 +51,16 @@
 
 @property (nonatomic, strong) NSString* labelDetail;
```

Comment:
*shrug* this is how the template that comes with mogenerator does it, and I only changed what I needed to change. For whatever reason the template adds a superfluous "readwrite" specifier to properties generated from relationships and not to those from attributes.